topics/transcriptomics/tutorials/clipseq/tutorial.md

```diff
@@ -0,0 +1,629 @@
+>    >
+>    {: .comment}
+>
+>    > ### {% icon comment %} Do the same thing for the input control data set.
+>    >
+>    > If you processed the RBFOX2 bam file then do the same thing for the input control bam or *vice verca*.
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. Why are we doing the de-duplication after the read mapping?
+> 2. What is the edit distance threshold?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. Because we need the coordinates of the reads to see if some reads have the same coordinates and the same UMI.
+> > 2. `UMI-tools dedulication` uses the hamming distance between UMIs to detect sequencing errors in the UMIs. A hamming distance of 1 means, that two UMIs are the same if they differ in maximal one character. If two characters are different, then these are two disparate UMIs.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Quality check of the de-duplication with **FastQC**
+
+> ### {% icon hands_on %} Hands-on: De-duplication quality control
+>
+> 1. **FastQC** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Short read data from your current history"*: `bam` (output of **UMI-tools deduplicate** {% icon tool %})
+> 2. View the **Sequence Duplication Levels** plot.
+> ![fastqafter](../../images/clipseq_duplication_level_2.png "Sequence duplication levels <b>after</b> de-duplication.")
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. What can you see, if you compare the duplication levels to our first quality control in step two?
+> 2. Why are the duplication levels still high?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. We have reduced the duplication levels.
+> > 2. The duplication levels are still high, because we don't have the full data sets (reads are missing).
+> {: .solution}
+>
+{: .question}
+
+# Step 6: Second Quality Control
+
+In this section we check the quality of our mapped reads and see if our samples are correlated or not. It is actually not really necessary for our data sets, because of the small sample size. However, for bigger analyses it is wise to check if some samples may encompass major quality problems.
+
+## Check the read coverage with **plotFingerprint**
+
+> ### {% icon hands_on %} Hands-on: Check the read coverage
+>
+> 1. **plotFingerprint** {% icon tool %} with the following parameters:
+>    - *"Sample order matters"*: `No`
+>    - *"Bam file"*: `both bam files` (output of **UMI-tools deduplicate** {% icon tool %})
+>    - *"Show advanced options"*: `yes`
+>     - *"Bin size in bases"*: `100`
+>    - *"Show advanced output settings"*: `no`
+> 2. View the output image.
+> ![fingerprint](../../images/clipseq_fingerprint.png "Graph of IP strength from <b>plotFingerprint</b>..")
```

Comment:
Could you describe the plot a little bit. I know you have it as Q, but the Q are more meant to self-check the trainee. You should explain before whats going before asking Q. Also maybe make clear that this is an exceptional good plot :)